eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -240,24 +240,34 @@ foreach ($packageDetail in $packageDetails) {
     }
 
     $attempt = 0
+    $success = $false;
+
     while ($attempt++ -lt 3) {
       Write-Information "Releasing staging repostiory $stagedRepositoryId, attempt $attempt"
       Write-Information "mvn org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:rc-release `"-DstagingRepositoryId=$stagedRepositoryId`" `"-DnexusUrl=https://oss.sonatype.org`" `"-DrepositoryId=target-repo`" `"-DserverId=target-repo`" `"-Drepo.username=$RepositoryUsername`" `"-Drepo.password=`"`"[redacted]`"`"`" `"--settings=$PSScriptRoot\..\maven.publish.settings.xml`""
       mvn org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:rc-release "-DstagingRepositoryId=$stagedRepositoryId" "-DnexusUrl=https://oss.sonatype.org" "-DrepositoryId=target-repo" "-DserverId=target-repo" "-Drepo.username=$RepositoryUsername" "-Drepo.password=""$RepositoryPassword""" "--settings=$PSScriptRoot\..\maven.publish.settings.xml"
 
       if ($LASTEXITCODE -eq 0) {
         Write-Information "Package $($packageDetail.FullyQualifiedName) deployed"
-        exit 0
+        $success = $true
+        break
       }
 
       Write-Information "Release attempt $attempt exited with code $LASTEXITCODE"
       Write-Information "Checking Maven Central to see if release was successful"
 
       if (Test-ReleasedPackage -RepositoryUrl $packageReposityUrl -PackageDetail $packageDetail) {
         Write-Information "Package $($packageDetail.FullyQualifiedName) deployed despite non-zero exit code."
-        exit 0
+        $success = $true
+        break
       }
     }
-    exit 1
+
+    if (!$success) {
+      Write-Information '##vso[task.logissue type=error]Release to Maven Central failed. For troubleshooting, see https://aka.ms/azsdk/maven-central-tsg'
```

Comment:
Should this be Write-Host now?

Reply:
I'm really not sure.   Write-Host is a wrapper for Write-Information.  We should probably replace all of our Write-Hosts with Write-Information as it's more why and less how.  I.e.  the difference between Write-Verbose and Write-Information is clearer than Write-Verbose and Write-Host.

Reply:
Except in this case, where the message is specifically for the ADO agent (host)